HousingRepairsOnlineApi/Helpers/RepairBookingResponseHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using Ardalis.GuardClauses;
+using HousingRepairsOnlineApi.Domain;
+
+namespace HousingRepairsOnlineApi.Helpers
+{
+    public class RepairBookingResponseHelper : IRepairBookingResponseHelper
+    {
+        private IRepairDayWindowHelper repairDayWindowHelper;
+
+        public RepairBookingResponseHelper(IRepairDayWindowHelper repairDayWindowHelper)
+        {
+            this.repairDayWindowHelper = repairDayWindowHelper;
+        }
+
+        public dynamic GetRepairBookingResponse(Repair repair)
+        {
+            if (repair.RepairType == RepairType.Communal)
```

Comment:
This conditional check doesn't feel right - can it be abstracted to different implementations of `IRepairBookingResponseHelper` each of which has logic specific to creating the correct response?

Each would need to be registered and resolved when required.